.github/workflows/bazel.yml

```diff
@@ -0,0 +1,35 @@
+name: Bazel build
+
+on:
+  pull_request:
+    paths-ignore:
+    # ignore markdown files (CHANGELOG.md, README.md, etc.)
+    - '**/*.md'
+  push:
+    paths-ignore:
+    # ignore markdown files (CHANGELOG.md, README.md, etc.)
+    - '**/*.md'
+    branches:
+    - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  tests:
+    name: Tests
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v3
+
+    - name: Build
+      run: bazel build :ngx_rust :ngx_wasm_rs_static :ngx_wasm_rs_shared --verbose_failures
+    
+    - name: Display artifacts
+      run: ls -lh bazel-bin/
```

Comment:
On second thought, we will need this to be a reusable workflow that we can invoke from `ci.yml` and `ci-large.yml`. Please follow the example with `job-unit-tests.yml` (or any of the `job-*.yml` reusable workflows we already have).

Reply:
I will leave this as a TODO for this PR once I finish the bazel part of it.